### DISCUSSION TITLE
Add a configured label to a PR if automerge is enabled for it

### DISCUSSION THREAD

**Discussion by dnwe** [renovatebot/renovate#24662] [Suggest an Idea]
### What would you like Renovate to be able to do?

Currently automerge can be enabled at various places in the config, as well as within packageRules for the different digest/patch/minor/major types of updates

```json
      "lockFileMaintenance": {
        "enabled": true,
        "automerge": true,
        "schedule": [
          "after 7am and before 8am on Mon,Tue,Thu,Wed and Fri"
        ]
      },
      "packageRules": [
        {
          "extends": [
            ":semanticCommitTypeAll(chore)"
          ],
          "automerge": true,
          "major": {
            "automerge": false
          },
          "matchDepTypes": [
            "devDependencies"
          ]
        },
        // etc.
```

It would be useful if a label could also be added to each PR to indicate that automerge has been enabled for it, such that it is visible on the PR list(s) and can also be filtered in/out via the `label:` search operator. Whilst (apart from lockFileMaintenance) it is _possible_ to achieve this within the existing config (with a lot of duplication of digest/patch/minor/major blocks amongst the existing automerge config), this could be simplified by having a global "automerge" label that renovate applied at creation/update time where it is already aware of whether automerge is going to be enabled or not for the given PR

### If you have any ideas on how this should be implemented, please tell us here.

- 

### Is this a feature you are interested in implementing yourself?

Maybe

**Comment by HonkingGoose** [comment]
> It would be useful if a label could also be added to each PR to indicate that automerge has been enabled for it, such that it is visible on the PR list(s) and can also be filtered in/out via the `label:` search operator. 

I see @viceice gave a 👎 to this feature request, but what do @rarkins and @JamieMagee think?

**Comment by rarkins** [comment]
I think it would be better implemented with a "tags" concept (Eg isAutomerge) and a new matchTags package rule concept. Then it can be extended without needing to bloat the config options every time. Eg we can use isBreaking, etc

**Comment by MPV** [comment]
I raised a separate/similar request for indicating in the onboarding PR and dashboard which PRs will be automerged:
- https://github.com/renovatebot/renovate/discussions/33168
